seed/index.js

```diff
@@ -45,13 +43,39 @@ module.exports = yeoman.generators.Base.extend({
     var done = this.async();
 
     // Have Yeoman greet the user.
-    this.log(yosay('Out of the box I include Cells\'s seed-element.'));
+    this.log(yosay('Out of the box I include a Cells\'s component.'));
 
     var prompts = [{
+        name: 'type',
+        message: 'Which type is your component?',
+        type: 'list',
+        choices:[{
+            name: 'ui-component',
+            value: 'ui-component',
+            checked: false
+          }, {
+            name: 'dp-component',
+            value: 'dp-component',
+            checked: false
+          }, {
+            name: 'dm-component',
+            value: 'dm-component',
+            checked: false
+          }]
```

Comment:
Maybe this induces to think that the component has to start with ui- as the DP and DM.
Could you change it to: 
name: 'UI (purely graphical)'....
name: 'DP (data provider)'..... 
name: 'DM (data manager)'